bundles/org.openhab.binding.meater/src/main/java/org/openhab/binding/meater/internal/handler/MeaterHandler.java

```diff
@@ -154,9 +156,9 @@ private State getValue(String channelId, Device meaterProbe) {
                 }
                 break;
             case CHANNEL_LAST_CONNECTION:
-                ZonedDateTime zdt = meaterProbe.getLastConnection();
-                if (zdt != null) {
-                    return new DateTimeType(zdt).toZone(timeZoneProvider.getTimeZone());
+                Instant instant = meaterProbe.getLastConnection();
+                if (instant != null) {
+                    return new DateTimeType(ZonedDateTime.ofInstant(instant, timeZoneProvider.getTimeZone()));
                 }
```

Comment:
Isn't a break missing just after this line ?

Reply:
Correct and fixed!